packages/plugin-ext/src/main/browser/view/plugin-view-widget.ts

```diff
@@ -131,8 +133,9 @@ export class PluginViewWidget extends Panel implements StatefulWidget {
 }
 export namespace PluginViewWidget {
     export interface State {
-        label: string
-        message?: string;
-        widgets: ReadonlyArray<Widget>
+        label: string,
+        message?: string,
+        widgets: ReadonlyArray<Widget>,
+        suppressVisibilityClause: boolean
```

Comment:
Should we rename it to match its actual value?:

```suggestion
        suppressUpdateViewVisibility: boolean
```

Reply:
I agree, this rename  is now reflected in the latest commit, thanks!